timed/reports/tests/test_notify_reviewers_unverified.py

```diff
@@ -27,14 +27,27 @@ def test_notify_reviewers(db, mailoutbox):
     ReportFactory.create(date=date(2017, 7, 1), task=task_no_work,
                          verified_by=reviewer_no_work)
 
-    call_command('notify_reviewers_unverified')
+    call_command(
```

Comment:
Please add tests for all call variants (with/without cc, with/without message)

Reply:
Done